### DISCUSSION TITLE
搜索引擎相关

### DISCUSSION THREAD

**Discussion by banned-historical-archives** [banned-historical-archives/banned-historical-archives.github.io#396] [Q&A] [answered]
是否有比google搜索更好的方案

**Comment by banned-historical-archives** [comment]
好像可以用github的搜索代替

https://github.com/banned-historical-archives/banned-historical-archives.github.io/search?q=邓小平

**Reply by banned-historical-archives** [reply]
但是不支持中文分词，很多内容搜不到

**Comment by PirateBook1** [comment]
好像没有什么好方法，我找到的方法不是付费（别的提供商的服务）就是需要自建服务器

**Comment by PirateBook1** [comment, marked as answer]
https://github.com/banned-historical-archives/banned-historical-archives.github.io/wiki/%E6%9C%AC%E5%9C%B0%E6%90%9C%E7%B4%A2%E7%9A%84%E4%BD%BF%E7%94%A8
